src/style.css

```diff
@@ -0,0 +1,120 @@
+* {
+  margin: 0;
+  box-sizing: border-box;
+}
+
+.content {
+  font-family: "Roboto", Arial, Helvetica, sans-serif;
+  font-size: 10px;
+  font-weight: 400;
+  line-height: 14px;

```

Comment:
don't create a separate `div`, just add this bem block to `body` tag. and remove font styles to `html` selector

Reply:
I removed font-styles to `html`, but styles which was in first block I can't remove to `body` tag because everything breaks in other tags and I removed it in first div class